tools/changelog/changelog.main.kts

```diff
@@ -381,40 +394,44 @@ fun extractReleaseNotes(body: String?, prNumber: Int, prLink: String): ReleaseNo
  *        JetBrains/compose-multiplatform-core
  */
 fun entriesForRepo(repo: String, firstCommit: String, lastCommit: String): List<ChangelogEntry> {
-    val pulls = (1..5)
+    val pulls = (1..10)
```

Comment:
It seems we start to lose PRs. There are too much :).

We seem lost these PRs for 1.8.0:
https://github.com/JetBrains/compose-multiplatform-core/pull/1550
https://github.com/JetBrains/compose-multiplatform-core/pull/1441